src/main/resources/partitions_statistics.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "t2.large",
```

Comment:
In current version it was the only one statistics I have. I decided to put all the knowledge it into code, so we can review it and select correct values just replacing configuration string. When we will get new statistics - we will just put it here.